### ISSUE TITLE
Category support seems to be missing

### ISSUE THREAD

**Issue by Yutao-Jin** [bill-ahmed/qbit-matUI#159]
**Is your feature request related to a problem? Please describe.**
 
Or did I miss it ?

**Describe the solution you'd like**

**Additional context**
Add any other context/links/screenshots about the feature request here.

**Comment by bill-ahmed** [comment]
Yup, the category column in torrent table is missing. There currently isn't any way to filter by category either, but that's something we could definitely add. I'll take a look when I get the chance :)
